pkg/csi/service/wcpguest/controller.go

```diff
@@ -307,6 +326,121 @@ func (c *controller) ControllerPublishVolume(ctx context.Context, req *csi.Contr
 	}
 	log.Debugf("Found virtualMachine instance for node: %q", req.NodeId)
```

Comment:
Why do we need to load the virtualMachine CR if the `ControllerPublishVolume` is for file volume? Dont we already know the VM name via `req.NodeId`?

Reply:
I think this is needed to watch on the specific resource version, see Line: 364